frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -310,13 +310,18 @@ export enum TerminationTypes {
 }
 
 export enum InsecureTrafficTypes {
+  // t('devconsole~None')
   None = 'None',
+  // t('devconsole~Allow')
   Allow = 'Allow',
+  // t('devconsole~Redirect')
```

Comment:
Dont see `Allow` and `Redirect` translations in the devconsole locales. We should be adding them.